tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1565,6 +1601,22 @@ schedule(void)
       } else {
         clear_rw_nodepchk();
       }
```

Comment:
The patch file here fails to apply. These three lines (1565-1567) were removed in PR #660 and replaced with other text.
```
@@ -1557,15 +1556,20 @@ schedule(void)
       merge_next_block = false;
     }

-    if (XBIT(183, 0x10000000)) {
-      if ((!XBIT(69, 0x100000)) && BIH_NODEPCHK(bih) &&
-          (!ignore_simd_block(bih))) {
-        fix_nodepchk_flag(bih);
-        mark_rw_nodepchk(bih);
-      } else {
-        clear_rw_nodepchk();
-      }
+    open_pragma(BIH_LINENO(bih));
+    BIH_NODEPCHK(bih) = !flg.depchk;
+    if(XBIT(19, 0x18))
+      BIH_NOSIMD(bih) = true;
+    else if(XBIT(19, 0x400))
+      BIH_SIMD(bih) = true;
+    if ((!XBIT(69, 0x100000)) && BIH_NODEPCHK(bih) &&
+        (!ignore_simd_block(bih))) {
+      fix_nodepchk_flag(bih);
+      mark_rw_nodepchk(bih);
+    } else {
+      clear_rw_nodepchk();
     }
+    close_pragma();

     for (ilt = BIH_ILTFIRST(bih); ilt; ilt = ILT_NEXT(ilt)) {
       if (BIH_EN(bih) && ilt == BIH_ILTFIRST(bih)) {
```